README.md

```diff
@@ -140,15 +140,15 @@ npm run publish-app
 - **How does Upscayl work?**
   - Upscayl uses AI models to enhance your images by guessing what the details could be. It uses Real-ESRGAN to achieve this.
 - **Is there a CLI available?**
-  - The CLI tool is called real-esrgan-ncnn-vulkan and it's available on the Real-ESRGAN repository.
+  - The CLI tool is called real-esrgan-ncnn-vulkan, and it's available on the Real-ESRGAN repository.
 - **Do I need a GPU for this to work?**
   - Yes, unfortunately. NCNN Vulkan requires a Vulkan-compatible GPU. Upscayl won't work with **most** iGPUs or CPUs. But hey, no harm in trying ;)
 - **How can I contribute?**
-  - You can report issues, fix code and add features by submitting PRs, or donate! 😊
+  - You can report issues, fix code, and add features by submitting PRs or donating! 😊
```

Comment:
You can’t add features by donating, and fixing code also needs submitting PRs. The original construction is intentional.